Client/Frontend/Browser/TabTrayController.swift

```diff
@@ -68,4 +270,51 @@ class TabTrayController: UIViewController, UITabBarDelegate, UITableViewDelegate
         tabManager.removeTab(tab)
         tableView.deleteRowsAtIndexPaths([indexPath], withRowAnimation: UITableViewRowAnimation.Automatic)
     }
-}
\ No newline at end of file
+}
+
+extension TabTrayController : Transitionable {
+    private func getTransitionCell(opts: TransitionOptions, browser: Browser?) -> CustomCell {
+        if let cell = opts["cell"] as? CustomCell {
+            cell.tab = browser
+            return cell
+        } else {
+            let cell = CustomCell(style: UITableViewCellStyle.Default, reuseIdentifier: "id")
+            cell.tab = browser
+            opts["cell"] = cell
+            return cell
+        }
+    }
+
+    func hideTransition(opts: TransitionOptions, vc: UIViewController) {
+        let container = opts["container"] as UIView
+
+        // Create a fake cell that is shown fullscreen
+        if let browser = vc as? BrowserViewController {
```

Comment:
Yeah. Originally I had all this transition code in its own separate class, but now that the view controllers handle the transitions themselves, we can just use them. Thanks :)